index.html

```diff
@@ -21,7 +21,7 @@ <h2>Code Journal</h2>
     </div>
   </header>
   <main>
-    <div data-view="edit-profile">
+    <div data-view="edit-profile" class="view" hidden>
```

Comment:
Your hidden class here is outside the quotes 😱

Reply:
This is an HTML attribute, not a class, I thought that was the way we were supposed to do this per the instructions.